dev/com.ibm.websphere.appserver.thirdparty.jsf-2.3/bnd.bnd

```diff
@@ -0,0 +1,91 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Fragment-Host: com.ibm.ws.org.apache.myfaces.2.3;bundle-version=1.0
+
+Bundle-Name: WebSphere JSF MyFaces Third Party API
+Bundle-SymbolicName: com.ibm.websphere.appserver.thirdparty.jsf-2.3
+Bundle-Description: WebSphere JSF 2.3 MyFaces Third Party API
+Implementation-Version: 2.3.3
+
+javac.source: 1.8
+javac.target: 1.8
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+# Don't export the org.apache.myfaces.buildtools package as that is only needed for compilation.  The jar that contains the 
+# org.apache.myfaces.buildtools package is javax.j2ee.jsf-2.2/lib/myfaces-builder-annotations-1.0.9.jar
+Export-Package: \
+  !org.apache.myfaces.buildtools.*, \
+  org.apache.myfaces.*;version=${Implementation-Version}, \
+  org.apache.myfaces.application;thread-context=true;mandatory:="thread-context";version=${Implementation-Version}, \
```

Comment:
Previously we were not exporting these at the implementation version were we? I assume moving forward we'll just export to match the version of the implementation? Previously we were at 1.0.17 I think?

Reply:
That's right - we were previously exporting all of the o.a.m. packages at `1.0.17`, which was probably arbitrarily chosen because it was the version at the time we worked on the feature.  Those o.a.m. packages weren't ever visible to applications, so it didn't matter.  Going forward (as the package is written now) every o.a.m. package will be exported with a version matching the myfaces impl level.